packages/medusa/src/loaders/pg-connection.ts

```diff
@@ -1,4 +1,4 @@
-import { ConfigModule } from "@medusajs/types"
+import { ConfigModule } from "@medusajs/framework"
```

Comment:
Maybe unrelated to the PR, but why is `ConfigModule` in `framework` and not `types`?

Reply:
For now I ve done that but both still exists. This will be better decided later on when we have a proper separation between what the framework provide and what medusa will do with it. Also, this loader will probably move to the framework as well so I thought it is fine for now